speedtest/datadog_checks/speedtest/speedtest.py

```diff
@@ -0,0 +1,158 @@
+from typing import Any, Dict
+import json
+import subprocess
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+MEGABYTE_TO_MEBIBYTE = 1048576.0/1000000.0
+
+
+class SpeedtestCheck(AgentCheck):
+    def check(self, instance):
```

Comment:
nit: The usage of instance in check method is deprecated. The recommended way is to use self.instance instead

Reply:
this was generated code from the instructions fwiw, I'd suggest updating the generators.
it also duplicates this in `check.py` when generating but I deleted this file

Reply:
Hi Cody - we updated our templates pretty recently (included in version 3.8.0), so probably came out after you created your initial skeleton: https://github.com/DataDog/integrations-core/pull/6673